setup.py

```diff
@@ -41,7 +43,7 @@
       test_suite='nose.collector',
       packages=['qp_shotgun', 'qp_shotgun/humann2'],
       package_data={'qp_shotgun': ['support_files/config_file.cfg']},
-      scripts=glob('scripts/*'),
+      scripts=['scripts/configure_shotgun', 'scripts/start_shotgun'],
```

Comment:
Part of the long set of tests to install metaphlan2.py, nothing in particular ...